SPECS/grub2/grub2.spec

```diff
@@ -214,18 +127,16 @@ Requires:       %{name}-configuration = %{version}-%{release}
 Minimal set of utilities to configure a grub-based system
 
 %prep
-# Remove module_info.ld script due to error "grub2-install: error: Decompressor is too big"
-LDFLAGS="`echo " %{build_ldflags} " | sed 's#-Wl,-dT,%{_topdir}/BUILD/module_info.ld##'`"
-export LDFLAGS
```

Comment:
I don't recall why this was added to the prep section. It is probably safe to remove, though, as long as no issues are seen during the build.

Reply:
Here's the full error:
```
WARN[0035] gpg: imported: 1 
WARN[0035] initrd generation of kernel 5.10.60.1-1.cm1 will be triggered later 
WARN[0035] initramfs (re)generation triggerin 1 1 2.0-8.cm1 
WARN[0035] initramfs 2.0-8.cm1 posttrans 
WARN[0047] Installing for i386-pc platform. 
WARN[0048] grub2-install: error: Decompressor is too big. 
INFO[0048] Flushing all IO to disk for /dev/loop0p2 
ERRO[0049] Failed to build image 
```

Reply:
related bug - https://bugs.funtoo.org/browse/FL-8245

Reply:
The image build did hit this error ("grub2-install: error: Decompressor is too big.") - https://dev.azure.com/mariner-org/mariner/_build/results?buildId=566907&view=logs&j=2ccbf1db-6c73-5af8-5ba2-cf05469dcf51&t=62118ca4-5f53-54b3-7ec9-8ef4ab0e9237&l=2789